exercises/isogram/src/Isogram.hs

```diff
@@ -0,0 +1,5 @@
+module Isogram (isIsogram) where
+
```

Comment:
I would remove one newline here so there is only one between `module` and `isIsogram`, not two.